### DISCUSSION TITLE
Treeview Height Limit

### DISCUSSION THREAD

**Discussion by whann0205** [Megabit/Blazorise#2081] [Q&A]
I am using smartphones as target of my application which limited space in x and y. 
By using Xamarin and mobile blazor bindings i have made quite a progress.
Latest development was using your treeview extension, which works very well. 
But that treeview is filled with a lot of data and 4 levels and therefore a lot of
space is consumed by that control.
I want to add a table based on the selection of that treeview,
The question is how can i limit the height of that treeview to be able to actually see the 
table or at least dont have to wipe for ages.
Maybe it would be better to put it right beneath the treeview but x space is also unlimited and the question is
if that looks better.
All thoughts welcome!
BTW since i am using xamarin i am stuck with version 0.9.3.2 because of  .net 3 :-(

**Comment by stsrki** [comment]
I guess you meant  0.9.2.x  of .net 3 :-)

Anyways, I think the best option is to use CSS and override `tree-view` class to define the width and height. There is no other way at this time.

**Reply by whann0205** [reply]
Hi stsrki,
Thanks for your help. Yep its 0.9.2.5 :-)
I have made a new css with the following lines: 
.tree-view {
    height: 30px;
    width: 20px;
    background-color: #ffd700;
}
That indeed work but as soon as the treeview gets populated that settings are ignored and the view is expanded again to hold all the content :-(
It seems as if the css is ignored then  - the area defined above is still visible with the defined
background-color(=orange) and the rest is a different one.
I am using Statehaschanged Function to redraw the treeview.
Any ideas?
br

**Reply by stsrki** [reply]
Try adding `overflow: auto;`

**Reply by whann0205** [reply]
Hi stsrki,
That did the trick now it works like it should. Great!
.tree-view {
    height: 40vh;
    width: 95%;
    background-color: #ffd700; 
    overflow: auto;
}
is the working one :-)

Last question is it possible to activate a scrollbar for that treeview?

**Reply by stsrki** [reply]
What do you mean by activating the scrollbar?

**Reply by whann0205** [reply]
I mean if that component has the ability to show a vertical scrollbar to show the user that content has much 
more entries than currently seen.

**Comment by whann0205** [comment]
Moved to mudblazor
